account_reconcile_oca/tests/test_bank_account_reconcile.py

```diff
@@ -999,6 +999,7 @@ def test_partner_name_with_parent(self):
         )
 
         self.invoice_matching_model.active = True
+        self.invoice_matching_model.match_text_location_label = False
```

Comment:
Why a separated commit just for this?

Reply:
It is already unified (with squash) in one commit, it was separated because it fixed an existing test error related to the change explained in the commit message.